app/scripts/views/reset_password.js

```diff
@@ -8,6 +8,7 @@ define(function (require, exports, module) {
   const AuthErrors = require('lib/auth-errors');
   const BaseView = require('views/base');
   const Cocktail = require('cocktail');
+  const FlowBeginMixin = require('views/mixins/flow-begin-mixin');
```

Comment:
This suggestion does open a can of worms though.

What happens if the user refreshes the page when they're on one of the password reset views? The server will have generated a fresh flow id and stuck it in the DOM then, and we won't have access to the old flow id any more.

Maybe it's enough of an edge case that we can forget about it for now, but longer term we might want to keep the flow id in session storage.